docs: update SQL Editor docs and screenshots (#763)
bytebase/bytebase.com: content/docs/sql-editor/admin-mode.md

In Admin Mode, you are not limited to run SELECT, DDL and DML change queries, but also allowed to run database administration SQL statements such as SHOW, SET, GRANT, etc.

![Admin Mode](/static/docs/sql-editor/admin-mode.webp)

> In Admin Mode, you can press <kbd>⌘ + Up / Down</kbd> or <kbd>Ctrl + Up / Down</kbd> to explore history statements backward or forward. Click the **Clear screen** button or use its keyboard shortcut <kbd>⇧ + ⌥ + C</kbd> to clear history queries on the screen.

Reviewer: ```suggestion
In Admin Mode, click the **Clear screen** button or use its keyboard shortcut <kbd>⇧ + ⌥ + C</kbd> to clear the screen. You can also press <kbd>⌘ + Up / Down</kbd> or <kbd>Ctrl + Up / Down</kbd> to explore history statements backward or forward. 
```
